README.md

```diff
@@ -519,6 +519,30 @@ func isError(error: Error?) -> Bool {
 }
 ```
 
+If there's a case where you want to enter a scope if AT LEAST ONE item is non-nil then use the `??` operator.
+
+```swift
+if let _ = item ?? item1 {
```

Comment:
IMHO this shouldn't be a standard, it's a good operator to save some lines but it doesn't provide more clarity to the code (it's even the opposite if you are not familiar with it). We shouldn't force developers to do it.